kem/kyber/templates/pkg.templ.go

```diff
@@ -5,10 +5,14 @@
 // Code generated from pkg.templ.go. DO NOT EDIT.
 
 // Package {{.Pkg}} implements the IND-CCA2 secure key encapsulation mechanism
-// {{.Name}}.CCAKEM as submitted to round 3 of the NIST PQC competition and
+{{ if .NIST }}
+// {{.KemName}} as submitted to round 3 of the NIST PQC competition and
 // described in
 //
 // https://pq-crystals.org/kyber/data/kyber-specification-round3.pdf
+{{ else }}
+// {{.KemName}} as defined in FIPS203.
```

Comment:
Non-NIST Kyber768.CCAKEM is not defined in FIPS203.

```suggestion
// {{.KemName}} as defined in
// https://pq-crystals.org/kyber/data/kyber-specification-round3.pdf
```

Reply:
Oops, swapped the branches around.